### DISCUSSION TITLE
How to retrieve unique size of ROI image

### DISCUSSION THREAD

**Discussion by ChengJungHsu** [kvos/CoastSat#434] [Q&A] [answered]
I found that some of polygon input may retrive two different sizes satellate images. Is it possible to avoid this issue otherwise it might cost double time in the next detection process. For example in the below setting, the images has two different pixel images, a smaller one is 607X402 pixels and another is 607X961 pixels. The smaller pixel image can be removed since it only contains incomplete information of ROI. And the other one whichs fits ROI is appropriate. Thank you advance for any suggestion and commnet.

Setting:
polygon = [[[120.1708287341472,23.34596326807398],
            [120.1708287341472,23.43096030303248],
            [120.1144107900542,23.43096030303248],
            [120.1144107900542,23.34596326807398],
            [120.1708287341472,23.34596326807398]]] 
dates = ['2022-12-01', '2023-01-01']
sat_list = ['S2']
collection = 'C02'

Two different size images:
![examplefig](https://github.com/kvos/CoastSat/assets/100653419/db7e3e9c-93dd-4e97-947a-4a04337be5ff)

**Comment by kvos** [comment, marked as answer]
hi @ChengJungHsu , yes it's annoying when it crops the ROI from two adjacent scenes. One way of overcoming this is to specify the tile, see https://github.com/kvos/CoastSat/pull/366

**Reply by ChengJungHsu** [reply]
Thank you! The trouble is fixed exactly. With additional keyword "S2tile" in input dictionary, coastsat retrives desired image within ROI. Following the issue #366 , the[ web page](https://sentinel.esa.int/web/sentinel/missions/sentinel-2/data-products) provides tiles information in kml file which is convient to check in google earth.

**Reply by kvos** [reply]
great, glad that you could fix it with the tile name.
